scripts/load_templates.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}"/.. || exit
+
+DESIGN_SYSTEM_VERSION="4.0.0"
+
+rm -rf /tmp/templates*
+curl -L --url "https://github.com/ONSdigital/design-system/releases/download/$DESIGN_SYSTEM_VERSION/templates.zip" --output /tmp/templates.zip
+unzip /tmp/templates.zip -d /tmp/templates
```

Comment:
This should unzip to a randomly generated directory name as we're forcefully removing it and there is always a chance that another app/service is using a directory of the same name. It looks like the `mktemp` command would be a good option - https://stackoverflow.com/questions/4632028/how-to-create-a-temporary-directory